R/clean_code.R

```diff
@@ -0,0 +1,151 @@
+utils::globalVariables(
+  c("cnt", "start", "impl", "lifetime", "type", "name")
+)
```

Comment:
I use {dplyr} & {tibble} to ease data manpulation, so I obviously get R CMD check complaints about undeclared variables.
I invoked `globalVariables` here, but if you prefer another solution to this problem, I will gladly implement it.

Reply:
See if you can resolve the issue using pronouns: https://rlang.r-lib.org/reference/tidyeval-data.html